docs/REST API/rest-api.md

```diff
@@ -4721,7 +4721,8 @@ Add a configuration profile to enforce custom settings on macOS and Windows host
 | ------------------------- | -------- | ---- | ------------------------------------------------------------------------------------------------------------- |
 | profile                   | file     | form | **Required.** The .mobileconfig and JSON for macOS or XML for Windows file containing the profile. |
 | team_id                   | string   | form | _Available in Fleet Premium_. The team ID for the profile. If specified, the profile is applied to only hosts that are assigned to the specified team. If not specified, the profile is applied to only to hosts that are not assigned to any team. |
-| labels                    | array     | form | _Available in Fleet Premium_. An array of labels to filter hosts in a team (or no team) that should get a profile. |
+| labels_include_all                    | array     | form | _Available in Fleet Premium_. Profile will only be applied to hosts that have all of these labels. |
```

Comment:
Add a second name for existing `labels` key: `labels_include_all`

Both keys have the same behavior: profile is only applied to hosts that have all labels.